ext/node/lib.rs

```diff
@@ -9,7 +9,10 @@ use deno_core::error::AnyError;
 use deno_core::located_script_name;
 use deno_core::op;
 use deno_core::serde_json;
+use deno_core::serde_v8;
 use deno_core::url::Url;
+#[allow(unused_imports)]
+use deno_core::v8;
```

Comment:
Remove?

Reply:
Needed for #[op(fast)] to work with serde_v8::Value